src/Microsoft.PowerShell.AccountManagement/commands/NewUserCommand.cs

```diff
@@ -0,0 +1,401 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Management.Automation;
+using System.DirectoryServices.AccountManagement;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The New-LocalUser cmdlet creates a new local user account.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "User",
+            DefaultParameterSetName = "Password",
+            SupportsShouldProcess = true,
+            HelpUri = "https://go.microsoft.com/fwlink/?LinkId=717981")]
+    public class NewUserCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Gets or sets a DateTime that specifies the date and time that the account expires.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true)]
+        [AllowNull]
+        public DateTime AccountExpires
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or sets a switch to specify that the account will not expire.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true)]
+        public SwitchParameter AccountNeverExpires
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or sets the type of store to which the account belongs.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true)]
+        public ContextType AccountStore
+        {
+            get;
+            set;
+        } = ContextType.Domain;
```

Comment:
Does it make sense to be Domain by default?  Does this work on non-Windows?

Reply:
It is only Windows API. Although it is based on "providers" and we could get support Unix PAM or something like in future.